socorro/siglists/irrelevant_signature_re.txt

```diff
@@ -17,6 +17,7 @@ dalvik-jit-code-cache
 dalvik-LinearAlloc
 dalvik-mark-stack
 data@app@org\.mozilla\.f.*-\d\.apk@classes\.dex@0x.*
+.*FNODOBFM.*
```

Comment:
I believe the `.*` at the end is unnecessary since `re.match` matches from the start of the line. But other lines in this file use it, maybe we should just be consistent. This is a nit anyway.

Reply:
Oh, funky--I didn't notice that. I'll nix the `.*` at the end.